### PR TITLE
[iOS] Change CALayer.Transform in MainThread

### LINKED ISSUE #1480: [iOS] _WebTryThreadLock error on WebView with TranslateTo
### Description

I am receiving a thread error on iOS when animating a WebView (with `TranslateTo`) inside a modal Page.

### Steps to Reproduce

1.  Open a page with `Navigation.PushModalAsync`
2. Inside this new Page, use `TranslateTo` on a WebView.

### Expected Behavior

WebView animates translation.

### Actual Behavior

App crashes, with the following error:

> bool _WebTryThreadLock(bool), 0x60400021ae80: Tried to obtain the web lock from a thread other than the main thread or the web thread. This may be a result of calling to UIKit from a secondary thread. Crashing now...

No stack trace is provided.

### Basic Information

- Version with issue: 2.5.0.12.19.34
- IDE: Visual Studio Community 7.3.2
- iOS: 11.6.1.2

### Reproduction Link

[ModalWebView.zip](https://github.com/xamarin/Xamarin.Forms/files/1594933/ModalWebView.zip)

### PR DESCRIPTION
**Pull request by AndreiMisiukevich**
### Description of Change ###

Confirm changing CALayer.Transform in Main Thread

### Issues Resolved ### 

- fixes #1480 

### API Changes ###
<!-- List all API changes here (or just put None), example:

Added:

Changed OnUpdateNativeControl of VisiualElementTracker

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###
Fixes bug

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

Test sample is attached to the issue

### CODE REVIEW

**Comment by rmarinho** [comment]
Can you add a UITest? or a sample in the galley?! Thanks

**Comment by AndreiMisiukevich** [comment]
@rmarinho yes, sure, i will do in 5-6 hours

**Comment by AndreiMisiukevich** [comment]
@rmarinho done. Added test case

**Comment by rmarinho** [comment]
@AndreiMisiukevich that's not really testing, Maybe add a label to the same page, and assert that label is there (it didn't crashed)

**Comment by AndreiMisiukevich** [comment]
@rmarinho i will set any source to WebView and add label to the top of the page

File: Xamarin.Forms.Platform.iOS/VisualElementTracker.cs [hunk 1]
```diff
@@ -260,7 +260,16 @@ void update()
 					transform = transform.Rotate(rotationY * (float)Math.PI / 180.0f, 0.0f, 1.0f, 0.0f);
 
 				transform = transform.Rotate(rotation * (float)Math.PI / 180.0f, 0.0f, 0.0f, 1.0f);
-				caLayer.Transform = transform;
+
+				if(Foundation.NSThread.IsMain)
```
Comment (samhouts) [inline Xamarin.Forms.Platform.iOS/VisualElementTracker.cs]:
```suggestion
				if (Foundation.NSThread.IsMain)
```

**Comment by AndreiMisiukevich** [comment]
@samhouts done. You may review again

**Comment by samhouts** [comment]
build --uitests

**Pull request merged**